index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script&family=Roboto:wght@700&family=Zen+Dots&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+    <main class="wrapper">
+    <header>
+        <p><a href="#">My Logo</a></p>
+    <nav class="image-logo">
+        <img src="images/Union.png" alt="navigation icon">
+    </nav>
+    </header>
+
+    <section class="section-one">
+        <h1>I'm Salma <br> Glad to see you!</h1>
+        <p>I'm a software developer!  
+            I can help you build a product , 
+            feature or website Look through 
+            some of my work and experience! 
+            If you like what you see and have
+             a project you need coded, 
+            don't hestiate to contact me.</p>
+            <button type="button" class="btn">LET'S CONNECT</button>
```

Comment:
- [ ] Kindly make sure to [remove the default style of the button](https://stackoverflow.com/questions/2460100/remove-the-complete-styling-of-an-html-button-submit#:~:text=button%2C%20input%5Btype%3D%22submit%22%5D%2C%20input%5Btype%3D%22reset%22%5D%20%7B%0A%09background%3A%20none%3B%0A%09color%3A%20inherit%3B%0A%09border%3A%20none%3B%0A%09padding%3A%200%3B%0A%09font%3A%20inherit%3B%0A%09cursor%3A%20pointer%3B%0A%09outline%3A%20inherit%3B%0A%7D) and add a style like the Figma design.

| Figma design | Your design| 
| --- | --- |
|![image](https://user-images.githubusercontent.com/83360636/212092068-fd3f5111-68b1-4c2e-a9e1-51d6796bef5b.png)  |![image](https://user-images.githubusercontent.com/83360636/212092164-cc34c0b8-f36e-4f59-83c8-5d007aaa9164.png)  |